assets/copy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+cd ../../gosling.js
```

Comment:
Would you mind adding descriptions on how to and when to use this for the development in README.md?